### ISSUE TITLE
Infinite loop on project loading

### ISSUE THREAD

**Issue by johannnovak** [godotengine/godot-docs#3400]
**Your Godot version:**
3.2.1 windows x64

**Issue description:**
A friend created a project on Linux with a steam downloaded version of GodotEngine (3.2.1)
On Windows, I downloaded on the official website of GodotEngine the software version 3.2.1.
I pulled his project from a git repository (which is almost bare, nothing much has been done inside).
The Godot ProjectManager launches correctly however when double-clicking on the imported project or selecting it and clicking on 'Launch', it closes the ProjectManager, opens a new window with the Godot splash screen and then after 5 seconds, does not respond anymore.

I tried downloading the sources to check where the issues comes from.
However it does not freeze like the runtime binary, it crashes instead ... 
I use VSCode to compile the sources.

Do you have an idea on the problem ?

**Comment by johannnovak** [comment]
Okay so after switching to Visual Studio Community to debug the thing, I found out that it is an OpenGL which freezes:
`glTexParameteri(texture->target, GL_TEXTURE_MAX_LEVEL, 0);`

In file **rasterizer_storage_gles3.cpp** line 1800, in method **RasterizerStorageGLES3::sky_set_texture**.

Callstack in joint file.
[godot_issue_callstack.txt](https://github.com/godotengine/godot-docs/files/4494807/godot_issue_callstack.txt)

I don't know if it is normal but it breaks twice in this method.

**Comment by johannnovak** [comment]
Okay so ! I figured what was the problem but I don't know why it freezes up the loading.

Leading to what I discovered before, I checked what was related to the skybox and it seems to be loading from the default environment (default_env.tres).
Here are the lines associated with the procedural sky :

```
[sub_resource type="ProceduralSky" id=1]
radiance_size = 5
ground_bottom_color = Color( 0.862745, 0.858824, 0.858824, 1 )
ground_horizon_color = Color( 1, 1, 1, 1 )
```

After checking on an empty project created (which is working by the way), here are the lines associated with the procedural sky :
`[sub_resource type="ProceduralSky" id=1]`

So one can tell that applying the radiance can cause freezes on startup ?

**Comment by Calinou** [comment]
> So one can tell that applying the radiance can cause freezes on startup ?

Yes, large radiance sizes will cause the graphics driver to abort on old/slow hardware. This is a known issue that will be solved in Godot 4.0.
